styles.css

```diff
@@ -2,377 +2,571 @@
+  background-size: cover;
+  background-color: #e5e5e5;
+  padding-bottom: 40%;
+  display: grid;
+  grid-template-columns: 1fr;
+  justify-content: center;
+}
 
-  #headerSection {
-    background-image: url("./media-library/headerbg1.png");
-    background-size: cover;
-    background-color: #e5e5e5;
-    padding-bottom: 40%;
-    display: grid;
-    grid-template-columns: 1fr;
-    justify-content: center;
-  }
+ul {
+  padding: 0;
+}
 
-  ul {
-    padding: 0;
-  }
+.links {
+  text-decoration: none;
+  color: #6070ff;
+}
 
-  .links {
-    text-decoration: none;
-    color: #6070ff;
-  }
+/* NAV SECTION */
+#navBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 10px 24px;
+}
 
-  /* NAV SECTION */
+.navItems {
+  flex: 50%;
+}
 
-  #navBar {
-    display: flex;
-    flex-direction: row;
-    justify-content: space-between;
-    align-items: center;
-    padding: 10px 24px;
-  }
+#logoContainer {
+  flex-direction: row;
+  align-items: center;
+}
 
-  .navItems {
-    flex: 50%;
-  }
+#logo {
+  height: 20px;
+  font-weight: 700;
+  font-size: 25px;
+  line-height: 20px;
+}
 
-  #logoContainer {
-    flex-direction: row;
-    align-items: center;
-  }
+#menuiconContainer {
+  text-align: right;
+}
 
-  #logo {
-    height: 20px;
-    font-weight: 700;
-    font-size: 25px;
-    line-height: 20px;
-  }
+/* ABOUT & ABOUT SELF SECTION */
+#aboutContainer {
+  background-size: cover;
+  height: 587.15px;
+  display: flex;
+  align-content: center;
+  flex-direction: column;
+  gap: 0;
+  padding: 10px 24px;
+  justify-content: center;
+}
+
+#aboutSelfContainer {
+  display: grid;
+  grid-template-columns: auto;
+  padding: 0 24px;
+  padding-top: 100px;
+  background-image: url("./media-library/Base.png");
+  height: 1050px;
+  background-size: cover;
+  background-color: #e5e5e5;
+}
+
+.aboutTitles {
+  font-weight: 700;
+  font-size: 35px;
+  line-height: 52px;
+  color: #172b4d;
+  width: 327px;
+  height: 190px;
+  padding-top: 60px;
+  padding-bottom: 0;
+}
+
+#aboutSelfTitle {
+  font-size: 30px;
+  line-height: 40px;
+  height: 20px;
+  padding-top: 0;
+  padding-bottom: 25px;
+}
+
+.summaries {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  width: 327px;
+  padding: -20px;
+}
+
+.socialsContainers {
+  color: #7f8cff;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.socialIcons {
+  padding-right: 17px;
+  text-decoration: none;
+  display: inline;
+}
+
+/* WORK SECTION */
+.workSection {
+  display: grid;
+  grid-template-columns: auto;
+  grid-row-gap: 88px;
+  padding: 114px 0;
+  background-color: #e5e5e5;
+  align-content: center;
+  align-items: center;
+}
+
+.workSamples {
+  background-color: #fff;
+  border-radius: 16px;
+  padding: 16px;
+  padding-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 327px;
+  border: 1px solid #dfe1e6;
+  margin: 0 auto;
+}
```

Comment:
- [ ] Kindly consider avoiding setting fixed `width` for your content when developing a responsive application. Setting fixed `width` for your content makes them only fit for specific screen sizes which isn't an ideal solution in responsive designs. To make your portfolio responsive across all screen sizes consider the below:
   - [ ] Remove the fixed width you have set on line `146` and adjust your styling so the design is responsive across `mobile, table`, and `desktop` screens.
  See below for more details:
<table>
<tr>
<th align=“middle”>Work section</th>
<th align=“middle”>About design</th>
</tr>
<tr>
<td><img width="675" alt="Screenshot 2023-05-12 at 10 35 47" src="https://github.com/geekelo/a-personal-portfolio-site/assets/107192076/0435270f-fb3a-448c-be0c-ed209c93b94e"></td>
<td>
<img width="585" alt="Screenshot 2023-05-12 at 10 38 52" src="https://github.com/geekelo/a-personal-portfolio-site/assets/107192076/cffb3958-4a1e-4098-aafe-e23d09aca4e5">
</td>
</tr>
</table>